x-pack/plugins/cases/public/components/user_action_tree/user_action_markdown.tsx

```diff
@@ -41,19 +41,21 @@ export const UserActionMarkdown = forwardRef<UserActionMarkdownRefObject, UserAc
     });
 
     const fieldName = 'content';
-    const { setFieldValue, submit } = form;
+    const { getFields, setFieldValue, submit } = form;
 
     const handleCancelAction = useCallback(() => {
       onChangeEditable(id);
     }, [id, onChangeEditable]);
 
     const handleSaveAction = useCallback(async () => {
       const { isValid, data } = await submit();
-      if (isValid) {
+      const fieldData = getFields();
+
+      if (isValid && fieldData.content.value !== content) {
```

Comment:
I believe you can check `data.content`:
```suggestion
      if (isValid && data.content !== content) {
```
so we don't need to use getFields

Reply:
Good point!

Reply:
oof... how did I miss that 🤦